docs/reference/cluster/http-stats.asciidoc

```diff
@@ -0,0 +1,121 @@
+[[cluster-https-stats]]
+=== HTTP stats API
+++++
+<titleabbrev>HTTP stats</titleabbrev>
+++++
+
+Returns cluster HTTP statistics.
+
+[[cluster-http-stats-api-request]]
+==== {api-request-title}
+
+`GET /_http_stats` +
+
+[[cluster-http-stats-api-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `monitor` or
+`manage` <<privileges-list-cluster,cluster privilege>> to use this API.
+
+
+[[cluster-http-stats-api-desc]]
+==== {api-description-title}
+
+You can use the cluster HTTP stats API to retrieve https statistics in a cluster.
+
+[role="child_attributes"]
+[[cluster-http-stats-api-response-body]]
+==== {api-response-body-title}
+
+`cluster_name`::
+(string)
+Name of the cluster. Based on the <<cluster-name>> setting.
+
+
+[[cluster-http-stats-api-response-body-http]]
+`http`::
+(object)
+Contains http statistics for the cluster.
++
+.Properties of `http`
+[%collapsible%open]
+======
+`current_open`::
+(integer)
+Current number of open HTTP connections for the cluster.
+
+`total_opened`::
+(integer)
+Total number of HTTP connections opened for the cluster.
+
+`clients`::
+(array of objects)
+Information on current and recently-closed HTTP client connections.
+Clients that have been closed longer than the <<http-settings,http.client_stats.closed_channels.max_age>>
+setting will not be represented here.
++
+.Properties of `clients`
+[%collapsible%open]
+=======
+`id`::
+(integer)
+Unique ID for the HTTP client.
+
+`agent`::
+(string)
+Reported agent for the HTTP client. If unavailable, this property is not
+included in the response.
+
+`local_address`::
+(string)
+Local address for the HTTP client.
```

Comment:
We probably ought to use the word "connection" instead of "client" in this section, right? The reason I think this is that (for example) this is the local address for the connection (it's the one on the elasticsearch server), but it's the remote connection for the client. Or maybe we ought to change this to `client_address` and `elasticsearch_address` or something?

Reply:
I just copied the same existent doc of `_nodes/_stats/http`, but I will update both

Reply:
Just checking out the whole section is called `clients` and provides information about them. For instance:

```
`opened_time_millis`::
(integer)
Time at which the client opened the connection.
```

so, changing to connection would imply deeper changes. 

About changing the fields, I think it's not possible because this is using the old `nodeStats` API, which already has these names.

Reply:
It makes sense that we can't change the fields. But we shouldn't just reuse documentation that's wrong, right? `local_address` is not the `Local address for the HTTP client.` It's the address that's local to the machine gathering the information, right? For example in the description for `request_size_bytes` we say `Cumulative size in bytes of all requests from this client.` So in that case we're using `client` to mean the thing making the requests. When we say `Local address for the HTTP client.`, are we also using the word `client` to mean "the thing receiving the connection from the other client"? I think at a minimum we ought to improve the descriptions for `local_address` and `remote_address` (and probably as a followup fix the documentation for the `_nodes` API, too).

Reply:
@masseyke both updated. Thanks!